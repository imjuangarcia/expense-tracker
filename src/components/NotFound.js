import React from "react";
import { Link } from "react-router-dom";

const NotFound = () => (
  <section className="not-found">
    404 - <Link to="/">Go home, you're drunk</Link>
  </section>
);

export default NotFound;
