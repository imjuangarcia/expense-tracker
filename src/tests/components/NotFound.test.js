import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../components/NotFound";

test("Should render the Not Found component", () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
