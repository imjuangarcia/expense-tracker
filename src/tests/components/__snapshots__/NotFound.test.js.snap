// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Not Found component 1`] = `
<section
  className="not-found"
>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go home, you're drunk
  </Link>
</section>
`;
